hawkular-inventory-itest-parent/hawkular-inventory-itest-dist/pom.xml

```diff
@@ -0,0 +1,592 @@
+  </parent>
+
+  <artifactId>hawkular-inventory-itest-dist</artifactId>
+
+  <properties>
+    <kettle.build.type>production</kettle.build.type>
+    <assembly.dir>${project.build.directory}/assembly</assembly.dir>
+    <wildfly.dist.zip.root.dir>wildfly-${version.org.wildfly}</wildfly.dist.zip.root.dir>
+    <hawkular.dist.zip.root.dir>${project.artifactId}-${project.version}</hawkular.dist.zip.root.dir>
+  </properties>
+
+  <build>
+    <plugins>
+      <!-- skip tests -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <skipTests>true</skipTests>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.hawkular.inventory</groupId>
+      <artifactId>hawkular-inventory-itest</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <!-- test dist dependencies -->
+    <dependency>
+      <groupId>org.hawkular.bus</groupId>
+      <artifactId>hawkular-bus-broker-wf-extension</artifactId>
+      <version>${test-dist-version.org.hawkular.bus}</version>
+      <classifier>module</classifier>
+      <type>zip</type>
+    </dependency>
+    <dependency>
+      <groupId>org.hawkular.bus</groupId>
+      <artifactId>hawkular-bus-ra-wf-module</artifactId>
+      <version>${test-dist-version.org.hawkular.bus}</version>
+      <classifier>module</classifier>
+      <type>zip</type>
+    </dependency>
+    <dependency>
+      <groupId>org.hawkular.nest</groupId>
+      <artifactId>hawkular-nest-wf-extension</artifactId>
+      <version>${test-dist-version.org.hawkular.bus}</version>
+      <classifier>module</classifier>
+      <type>zip</type>
+    </dependency>
+    <dependency>
+      <groupId>org.hawkular.inventory</groupId>
+      <artifactId>hawkular-inventory-dist</artifactId>
+      <version>${test-dist-version.org.hawkular.inventory}</version>
+      <type>war</type>
+    </dependency>
+    <!--<dependency>-->
+      <!--<groupId>org.hawkular.cmdgw</groupId>-->
+      <!--<artifactId>hawkular-command-gateway-war</artifactId>-->
+      <!--<version>${version.org.hawkular.cmdgw}</version>-->
+      <!--<type>war</type>-->
+    <!--</dependency>-->
+    <!--<dependency>-->
+      <!--<groupId>org.hawkular.metrics</groupId>-->
+      <!--<artifactId>hawkular-metrics-api-jaxrs</artifactId>-->
+      <!--<version>${version.org.hawkular.metrics}</version>-->
+      <!--<type>war</type>-->
+    <!--</dependency>-->
+    <dependency>
+      <groupId>org.hawkular.commons</groupId>
+      <artifactId>hawkular-commons-embedded-cassandra-ear</artifactId>
+      <version>${test-dist-version.org.hawkular.commons}</version>
+      <type>ear</type>
+    </dependency>
+    <dependency>
+      <groupId>org.hawkular.accounts</groupId>
+      <artifactId>hawkular-accounts</artifactId>
+      <version>${test-dist-version.org.hawkular.accounts}</version>
```

Comment:
same as bus. I.e. here^ as well as in rest-api https://github.com/hawkular/hawkular-inventory/blob/master/hawkular-inventory-rest-api/pom.xml#L106 , `${version.org.hawkular.accounts} should be used that in turn should be defined here around here in top leve inv parent: https://github.com/hawkular/hawkular-inventory/blob/master/pom.xml#L56

I see no reason for using `test-dist-` prefix even for `commons` you use it and you depend on it. No matter that it is in the test. There is only one convention for version prop names - they should start with `version.org.hawkuar...`